services/base-service/models/expert/api.js

```diff
@@ -36,27 +63,37 @@ async function sanitize(req, res) {
         delete doc["@graph"][i]["totalAwardAmount"];
       }
     }
-    res.status(200).json(doc);
+    return doc;
   }
 }
 
 // this path is used instead of the defined version in the defaultEsApiGenerator
 router.get(
-  '/*',
+  '/expert/*',
   async (req, res, next) => {
+
+    const acceptHeader = req.headers.accept;
+
     logger.info(`GET ${req.url}`);
-    let id = '/'+model.id+decodeURIComponent(req.path);
+    let id = '/' + model.id + decodeURIComponent(req.path);
     try {
       let opts = {
-        admin : req.query.admin ? true : false,
+        admin: req.query.admin ? true : false,
       }
       res.thisDoc = await model.get(id, opts);
-      next();
-    } catch(e) {
+      // next();
+    } catch (e) {
       res.status(404).json(`${req.path} resource not found`);
     }
+    let doc = await sanitize(req, res);
+    doc = res.thisDoc;
```

Comment:
Isn't this better as a function next() after the sanitize step? like a `reformat` or `Accept` step?

Reply:
@jrmerz can you weigh in here?

Reply:
```javascript
router.get('/expert/*', async (req, res, next) => {
    // JM - this should be a global middleware if you want logging
    // logger.info(`GET ${req.url}`);

    let id = '/' + model.id + decodeURIComponent(req.path);
    try {
      let opts = {
        admin: req.query.admin ? true : false,
      }
      res.aeResponse = await model.get(id, opts);
      next();
    } catch (e) {
      return res.status(404).json(`${req.path} resource not found`);
    }
  }, 
  sanitize, 
  siteFarmFormat, // JM - this could send the response as well
  (req, res) => {
    res.status(200).json(res.aeResponse); 
  }
)
```